website_sale_product_pack/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Sale Product Pack",
+    "category": "E-Commerce",
+    "summary": "Compatibility module of product pack with e-commerce",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "depends": ["website_sale", "sale_product_pack"],
```

Comment:
Shouldn't this depend on `website_sale_stock`?

Reply:
A pack of services would be valid as well